### PR TITLE
Feat/event details bug

### LINKED ISSUE #110: EventScraper does not consistently scrape event.imageUrl
**Describe the bug**
For many events the event scraper does not scrape the url associated with the event image. Instead it scrapes something like `data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7`

**To Reproduce**
Steps to reproduce the behaviour:
1. Start firebase locally with `firebase emulators:start`
2. Call scrapeEventGivenCity cloud function locally with a given `city` as input
3. Check local firebase > events > scraped-events > `city`
4. the imageUrl attribute is not the correct value for many events

**Expected behaviour**
The scraper should scrape all event urls correctly

### PR DESCRIPTION
**Pull request by lx1922**
### Issue
Closes #110 

### What is changed / added
- parsing date now to avoid extra text from the scraping
- parsing imageURL to avoid image url coming out as some random base64 string

### Screenshots
-

https://user-images.githubusercontent.com/43979159/143367252-90756cbf-707b-4bf8-9e56-ee021eea2a0f.mp4

### CODE REVIEW

**Comment by lx1922** [comment]
Added @dark7866 since now scraper affects application

File: src/platform/functions/index.js [hunk 1]
```diff
@@ -68,14 +70,28 @@ const scrapeCityEvents = async (city) => {
...
+        if (eventDate) {
+          if (eventDate.innerText.indexOf('+') !== -1) {
+            parsedDate = eventDate.innerText.substring(0, eventDate.innerText.indexOf('+') - 1);
+          } else {
+            parsedDate = eventDate.innerText;
+          }
+        }
+
+        if (imgUrl) {
+          if (!imgUrl.getAttribute('src').includes('data:image/gif;base64')) {
```
Comment (TheFatPanda97) [inline src/platform/functions/index.js:82]: magical

**Review by TheFatPanda97 — APPROVED** [review]
LGTM

File: src/platform/functions/index.js (hunk 1 above)
Comment (lx1922) [reply src/platform/functions/index.js:82]: haha :) I think its something to do with the ternary operators and storing the string in a variable?

**Pull request merged**